src/interfaces/IMetaMorpho.sol

```diff
@@ -7,8 +7,8 @@ import {IERC4626} from "@openzeppelin/interfaces/IERC4626.sol";
 struct MarketConfig {
     /// @notice The maximum amount of assets that can be allocated to the market.
     uint192 cap;
```

Comment:
Then we could extend this type up to `uint248`

Reply:
I don't think we should do it because pending cap is used as an `uint192` everywhere. And we can't change it easily because it is also used in `PendingUint192`